.github/workflows/validate-consuming.yaml

```diff
@@ -0,0 +1,40 @@
+on:
+  pull_request:
+    types: [labeled, opened, synchronize, reopened]
+
+name: Validations (Consuming Projects)
+jobs:
+  validate:
+    timeout-minutes: 10
+    runs-on: ubuntu-latest
+    if: |
+      ( github.event.action == 'labeled' && github.event.label.name == 'validate-projects' )
+      || ( github.event.action != 'labeled' && contains(github.event.pull_request.labels.*.name, 'validate-projects') )
+    strategy:
+      fail-fast: false
+      matrix:
+        project: ['admiral', 'submariner', 'submariner-operator', 'lighthouse']
+    steps:
+      - name: Checkout the Shipyard repository
+        uses: actions/checkout@master
+
+      - name: Reclaim free space!
+        run: |
+          sudo swapoff -a
+          sudo rm -f /swapfile
+          df -h
+          free -h
+
```

Comment:
```suggestion
```

This part is only needed for E2E, not necessary for validations/unit tests and such things

Reply:
Removed, thanks!